### PR TITLE
Add digest_md5

### LINKED ISSUE #71: Add MD5 hashing
## Task Summary

📅 **Due date:** N/A  
🎯 **Success criteria:** `digest_md5` function in `file_util.py`  

Similar to `digest_sha256` function. Came from a comment in a PR, see https://github.com/starlinglab/starling-integrity-api/pull/64#discussion_r826944668

### PR DESCRIPTION
**Pull request by makeworld-the-better-one**
Closes #71

This also updates the other digest function to use a 32 KiB buffer instead of the 4 KiB one. This should improve hashing speed. This specific number is taken from Go stdlib, where `io.Copy` uses a 32 KiB buffer if needed. The perfect buffer size for a Linux system varies, but this should be good enough, and better than before.

### CODE REVIEW

File: starlingcaptureapi/file_util.py
Comment (anaulin) [inline starlingcaptureapi/file_util.py:63]: I would extract the magic block size number into a module-level constant, especially since it is reused.

File: starlingcaptureapi/file_util.py
Comment (anaulin) [inline starlingcaptureapi/file_util.py:67]: This function seems identical to `digest_sha256`, except for setting a different `hasher`. I would create a more general function `digest` that takes a parameter to indicate either `sha256` or `md5` algorithm that then picks the correct "hasher".

If we don't want the callers to have to use the more general interface, we can add one-line convenience functions `digest_md5` and `digest_sha256` that call the more general implementation with the correct algorithm.

Comment (makeworld-the-better-one) [reply starlingcaptureapi/file_util.py:94]: Done in dfb9b23

**Comment by makeworld-the-better-one** [comment]
@anaulin all your comments should now be addressed. Will wait for another review like from @benhylau before merging.

**Review by benhylau — APPROVED** [review]

**Pull request merged**
